src/cow-react/modules/wallet/web3-react/connection/ledger.tsx

```diff
@@ -5,26 +5,34 @@ import { useIsActiveWallet } from 'hooks/useIsActiveWallet'
 
 import { ConnectWalletOption } from '@cow/modules/wallet/api/pure/ConnectWalletOption'
 import { default as LedgerImage } from '@cow/modules/wallet/api/assets/ledger.svg'
-import { Ledger } from '../connectors/LedgerConnector'
 import { initializeConnector } from '@web3-react/core'
 import { Web3ReactConnection } from '../types'
 import { RPC_URLS } from 'constants/networks'
+import { AsyncConnector } from './asyncConnector'
 
 const BASE_PROPS = {
   color: '#4196FC',
   icon: LedgerImage,
   id: 'ledger',
 }
 
-const [ledger, ledgerHooks] = initializeConnector<Ledger>(
+const [ledger, ledgerHooks] = initializeConnector<AsyncConnector>(
   (actions) =>
-    new Ledger({
-      actions,
-      options: {
-        rpc: RPC_URLS,
-      },
-    })
+    new AsyncConnector(
+      () =>
+        import('../connectors/LedgerConnector').then(
```

Comment:
Please, make sure that it's only import from `LedgerConnector/index.ts` file, otherwise lazy-loading won't work